source/_integrations/python_script.markdown

```diff
@@ -128,6 +128,14 @@ The above `python_script` can be called using the following YAML as an input.
     rgb_color: [255, 0, 0]
 ```
 
+Introduced in Home Assistant [2023.7](/blog/2023/07/05/release-20237/), services can respond with data. Retrieving this data in your Python script can be done by setting the 'blocking' and 'return_response' arguments of the 'hass.services.call' function to 'True'. This is shown in the example below, in this case retrieving the weather forecast and putting it into a variable:
```

Comment:
This is documentation, not a changelog. Let's remove the mention of when it was introduced and the release notes link.

Reply:
Thanks for the feedback, I've removed the mention.